sci-electronics/gazebo/gazebo-2.2.1.ebuild

```diff
@@ -40,13 +40,14 @@ DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen app-text/ronn )
 "
 
-PATCHES=(
-	"${FILESDIR}/${P}-stdint.patch"
-)
-
-SRCDIR="$(ls ${WORKDIR} | grep osrf-gazebo | tail -n 1)"
+SRCDIR="osrf-gazebo-3b8a07b268a5"
```

Comment:
`osrf-gazebo-3b8a07b268a5` is the name of the folder after the unpack

And I get this error

```
 * QA Notice: 'grep' called in global scope: sci-electronics/gazebo-2.2.1
ls: cannot access '/var/tmp/portage/sci-electronics/gazebo-2.2.1/work': No such file or directory
```

Manually gives me

```
ls /var/tmp/portage/sci-electronics/gazebo-2.2.1/work/
osrf-gazebo-3b8a07b268a5
```